pkg/report/report.go

```diff
@@ -49,9 +49,18 @@ type Ruleset struct {
 
 // Rule contains information about a ran rule.
 type Rule struct {
-	ID     string  `json:"id"`
-	Name   string  `json:"name"`
-	Checks []Check `json:"checks"`
+	ID       string             `json:"id"`
+	Name     string             `json:"name"`
+	Severity rule.SeverityLevel `json:"severity,omitempty"`
+	Checks   []Check            `json:"checks"`
+}
+
+// CreateRuleTitle creates a title string that is used for describing the rules in the HTML reports
+func CreateRuleTitle(id string, severity rule.SeverityLevel, name string) string {
```

Comment:
```suggestion
// ruleTitle returns a rule title string with the given id, severity and name.
func ruleTitle(id string, severity rule.SeverityLevel, name string) string {
```
No need to export the function, this function should be lower in the file near functions like `rulesetSummaryText` and `rulesWithStatus`